### DISCUSSION TITLE
Can I squash all commit history into one to reduce repo size?

### DISCUSSION THREAD

**Discussion by liblaf** [upptime/upptime#720] [Q&A] [answered]
As stated in the title.
The size of the repo will increase over time. I found that this is mainly due to the large size of the graphs. If I delete all the history and only keep the latest records, will it work as normal?

**Comment by AnandChowdhary** [comment, marked as answer]
It should continue to work, yes, but your graphs and history will disappear.

**Reply by liblaf** [reply]
Is it necessary to keep the history of graphs? `upptime` seems not to depend on the history of graphs.

**Reply by AnandChowdhary** [reply]
No, it's not necessary for Upptime to work, but on your status page website, historic graphs won't show up for individual sites.

**Reply by liblaf** [reply]
Thanks.
